src/react/useReadyForPrint.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+import { describe, expect, test } from 'vitest';
+import { useReadyForPrint } from './useReadyForPrint';
+
+const IS_READY_CONTAINER = 'isReadyContainer';
```

Comment:
nit: would use kebab case for the data-testid, like we do in guideline-blocks
```suggestion
const IS_READY_CONTAINER = 'is-ready-container';
```